exec/java-exec/src/main/java/org/apache/drill/exec/store/LocalSyncableFileSystem.java

```diff
@@ -141,11 +141,11 @@ public LocalSyncableOutputStream(Path path) throws FileNotFoundException {
       output = new BufferedOutputStream(fos, 64*1024);
     }
 
-    @Override
-    public void sync() throws IOException {
+
+   /* public void sync() throws IOException {
```

Comment:
`LocalSyncableOutputStream` should implement `Syncable#sync()` to avoid compilation error. When sync will be removed, it can be changed.